$NetBSD: patch-ad,v 1.1 2013/04/24 02:20:38 rodent Exp $

Don't build the programs - use the ones from the base system.

--- Makefile.orig	2005-05-26 01:57:02.000000000 +1200
+++ Makefile
@@ -287,7 +287,7 @@ showversion:	ispell
 	  mv y_tab.o $@; \
 	  rm -f y_tab.c
 
-all-languages:	munchable
+all-languages:
 	$(MAKE) LANGUAGE_TARGET=all SHELLDEBUG=$(SHELLDEBUG) language-subdirs
 
 install: config.sh all install-basic install-deformatters
